### PR TITLE
chore(ci): use github sha instead of merge request event

### PR DESCRIPTION
**Pull request by fargito**
I'm not sure this works at all...

### CODE REVIEW

**Comment by fargito** [comment]
It doesn't work, see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

**Review by guillaumeduboc — APPROVED** [review]
lgtm

File: .github/workflows/pr-check-examples.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ name: 🔍 PR Check examples
 env:
   CI: true
   NODE_VERSION: 16
+  TARGET_SHA: ${{ github.event.pull_request.head.sha }} || ${{ github.event.merge_group.head_sha }}
```
[the remark this answers is not in the document]
Comment (guillaumeduboc) [reply .github/workflows/pr-check-examples.yml:27]: isn't `github.sha` doing this for us?

[the remark this answers is not in the document]
Comment (fargito) [reply .github/workflows/pr-check-examples.yml:27]: No, in fact for the pull request it contains the sha of the last commit on the target branch

**Pull request merged**